### PR TITLE
Feature/148069-Page-View-for-PowerBi

### PR DESCRIPTION
**Pull request by Noggling**
# Description

Added the page viewer.  can display custom compenent and power bi from fusion at the moement.

Completes: [AB#148256](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/148256)  [AB#148069](https://dev.azure.com/Equinor/fa63ceea-8883-41e2-8823-a3b54e4be178/_workitems/edit/148069)

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/Core/PageViwer/Components/PageViewer/PageViewer.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { PageViewWrapper } from './PageViewerStyles';
+
+export function PageViewer(props: ViewState): JSX.Element {
+    const [activePage, setActivePage] = useState(0);
+
+    const handleChange = (index: number) => {
+        setActivePage(index);
+    };
+
+    if (Object.values(props.pages).length === 0) return <NoPages />;
```
Comment (Gustav-Eikaas) [inline src/Core/PageViwer/Components/PageViewer/PageViewer.tsx:16]: if props.pages.length = 1? Still show tabs?

File: src/Core/PageViwer/Context/PageViewerContextProvider.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { PropsWithChildren, useReducer } from 'react';
+import { Context } from './PageViewerContext';
+import { pageViewerReducer } from './PageViewerReducer';
+
+interface PageViewerContextProviderProps extends PageViewerState {
```
Comment (Gustav-Eikaas) [inline src/Core/PageViwer/Context/PageViewerContextProvider.tsx:5]: temp?

**Review by Gustav-Eikaas — APPROVED** [review]
LGTM

File: src/Core/PageViwer/Context/PageViewerContextProvider.tsx (hunk 2 above)
Comment (Noggling) [reply src/Core/PageViwer/Context/PageViewerContextProvider.tsx:5]: hehe, I shall look at it ;)

File: src/Core/PageViwer/Components/PageViewer/PageViewer.tsx (hunk 1 above)
Comment (Noggling) [reply src/Core/PageViwer/Components/PageViewer/PageViewer.tsx:16]: Will look in to it ;)

**Pull request merged**